.github/workflows/docs.yml

```diff
@@ -41,13 +41,15 @@ jobs:
       run: echo "$HOME/.local/bin" >> $GITHUB_PATH
 
     - name: Build the documentation
-      working-directory: scripts
-      run: python3 generate_docs.py
+      run: |
+        mkdir build
+        cd build
+        python3 ../docs/generate_docs.py
 
     - name: Upload artifact
       uses: actions/upload-pages-artifact@0252fc4ba7626f0298f0cf00902a25c6afc77fa8 # v3.0.0
       with:
-        path: docs/html
+        path: build/docs_build/generated/html
```

Comment:
If we are to stick with the `build` directory for generated documentation, I think that the subdirectory `docs_build` could be renamed to `docs`

Reply:
the script can be run from anywhere, so if I run this e.g. from `docs` dir, it would try to add a new dir `docs` when there's already one...